client-rest/package.json

```diff
@@ -12,41 +12,39 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.1.0",
-    "@angular/common": "^6.1.0",
-    "@angular/compiler": "^6.1.0",
-    "@angular/core": "^11.0.5",
-    "@angular/forms": "^6.1.0",
-    "@angular/http": "^6.1.0",
-    "@angular/platform-browser": "^6.1.0",
-    "@angular/platform-browser-dynamic": "^6.1.0",
-    "@angular/router": "^6.1.0",
-    "core-js": "^2.5.4",
+    "@angular/animations": "11.2.14",
+    "@angular/common": "11.2.14",
+    "@angular/compiler": "11.2.14",
+    "@angular/core": "11.2.14",
```

Comment:
whoa, that's a big jump in angular versions! Nice

> I tested by building and running a docker image. I didn't do a full transfer, but tested the job creation flow.

Your dev console didn't show any new JS errors throughout this?

Reply:
Nope, no JS errors.